### DISCUSSION TITLE
LocalizedContentItemIndex for published content remains after draft changes are saved.

### DISCUSSION THREAD

**Discussion by rolandgee** [OrchardCMS/OrchardCore#10619] [Q&A] [answered]
Hi all,

After upgrading from rc2 to v1.0, I noticed that when saving draft changes to a published page, instead of only keeping the Localized Content Item Index for the latest version, a new one is added for the draft. Looking at the changes in Localized Content Item Index Provider, it may be from this PR #8515.

Is this a feature? My team has built some custom logic on top which trusts that there can only be one Localized Content Item Index per content item id. If that is not the case, then we will have to fix our code :) .

Thank you for your time!

**Comment by Skrypt** [comment, marked as answer]
You can filter these records simply by using Published and Latest columns.
If you only want the latest published content item you do `Published = 1 AND Latest = 1`.
If you only want the latest drafted content item you do `Published = 0 AND Latest = 1`
This is because of a change in the IndexingProvider which allows to also see drafted Localized content items which was necessary from the admin.

**Reply by rolandgee** [reply]
Ah, thanks for answer and rationale!
